esf_main.tex

```diff
@@ -291,18 +332,37 @@ \subsubsection{Position in car/mechanical fastening/mechanical connection}
 %Provide CAD-renderings showing all relevant parts and discuss the mechanical connection of the sensors to the pedal assembly. Mark the parts in the rendering, if necessary.
 TODO
 
+
 \subsection{Reset / Latching for IMD and BMS}\label{reset_latching_for_imd_and_bms}
 \subsubsection{Description/circuitry}
 %Describe the concept and circuitry of the latching/reset system for a tripped IMD or BMS.  Describe the method for resetting the IMD and BMS.
-TODO
+%TODO
+
+Resetting the IMD latch, which is shown in figure~\ref{fig:IMDLatch_Schematic} , requires restarting the GLV system. Restarting the GLV system will cause the IMD latch to send a pulse. If the output of the IMD is high because there is no ground fault, the pulse from the GLV activation will close the secondary relay, which allows the signal from the IMD to close the shutdown circuit.
+
+
+% find out details of BMS
 
 \subsubsection{Wiring/cables/connectors}
 %Describe wiring, show schematics, describe connectors and cables used and show useful data regarding the wiring.  If not detailed in section 2.1, be sure to show how the device opens the shutdown circuit.
-TODO
+%TODO
+
+The IMD latch will be connected to the ?? board via a 4-pin Molex LLC Ultrafit 1722861104 with 20 AWG wire, and the ?? board using a  6-pin TE Connectivity MICRO MATCH 338068-6 connector with 28 AWG ribbon cable. The 20 AWG wire is used for the shutdown circuit and to supply power and ground to the IMD latch. The ribbon cable carries signals from the IMD.%On the ?? board it will connect , or another Molex product of the same series but with a different number of headers on each side of the connector.
```

Comment:
Are we using the wiring found here? 
Mouser Part #:
571-19711119
Manufacturer Part #:
1-971111-9

http://www.mouser.com/ProductDetail/TE-Connectivity/1-971111-9/?qs=tNpDSWKagOPkPLjfSGNYAA%3d%3d